src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -924,6 +929,37 @@ public Unit getStrategicBombingRaidTarget(final Territory territory, final Colle
     return selected.get();
   }
 
+  @SuppressWarnings("serial")
+  class UnitRenderer extends JLabel implements ListCellRenderer<Unit> {
```

Comment:
This class is only used within `TripleAFrame` and can thus be private.  That would also allow you to give the constructor default accessibility.

Reply:
Changed to private